### ISSUE TITLE
Add Time stamp to Output

### ISSUE THREAD

**Issue by DisDoh** [godotengine/godot-proposals#1786]
**Describe the project you are working on:**
Working on Web/BlueTooth Poker 8
**Describe the problem or limitation you are having in your project:**
Would be helpful to see the timestamp in the ouptut console of Godot,
To compare some data when the have been printed in the output console.
**Describe the feature / enhancement and how it helps to overcome the problem or limitation:**
It could help for example the last build was and you just have to look at the timestamp to see it.
But i have to admit it's quite fancy so not a really improvement but a detail improvement.
**Describe how your proposal will work, with code, pseudocode, mockups, and/or diagrams:**
The timestamp could be in this format:
hh:mm-DD-MM-YYYY : Output log
**If this enhancement will not be used often, can it be worked around with a few lines of script?:**
no
**Is there a reason why this should be core and not an add-on in the asset library?:**
yes , it's console related so it should be core.

**Comment by Calinou** [comment]
The Errors tab already has timestamps for every message, but the Output tab doesn't have timestamps yet.

**Comment by and3rson** [comment]
Is there a way to enable timestamps in log files? I couldn't find one.

**Comment by DisDoh** [comment]
It's not yet implemented and it hasn't been picked to be implemented yet.
You can upvote this feature request and maybe it would be selected.

**Comment by Calinou** [comment]
> Is there a way to enable timestamps in log files? I couldn't find one.

No, but if you use your own logging, you can add timestamps when printing. This won't affect built-in error/warning prints though.
